### PR TITLE
Use cross-platform symbols

### PR DESCRIPTION
**Pull request by kevgo**
@charlierudolph 

Some Windows shells only support a limited character set. Using [figures](https://github.com/sindresorhus/figures) makes the new unicode output of Cucumber more compatible across all platforms.

### CODE REVIEW

File: lib/cucumber/listener/pretty_formatter.js [hunk 1]
```diff
@@ -16,9 +17,9 @@ function PrettyFormatter(options) {
   };
 
   var characters = {};
-  characters[Cucumber.Status.AMBIGUOUS] = '✖';
-  characters[Cucumber.Status.FAILED] = '✖';
-  characters[Cucumber.Status.PASSED] = '✓';
+  characters[Cucumber.Status.AMBIGUOUS] = figures.warning;
```
Comment (charlierudolph) [inline lib/cucumber/listener/pretty_formatter.js]: Please keep this as `figures.cross`. Failed and ambiguous are both treated as "Failures" in the summary output so I'd like them to use the same symbol

**Comment by charlierudolph** [comment]
Very cool. Thanks for this @kevgo. I want to setup appveyor anyway so we can make sure things are working for windows.

File: lib/cucumber/listener/pretty_formatter.js (hunk 1 above)
Comment (kevgo) [inline lib/cucumber/listener/pretty_formatter.js]: done

**Comment by kevgo** [comment]
All comments addressed

**Comment by kevgo** [comment]
Btw please check out https://github.com/Originate/exo-add for a working example of a Node code base on AppVeyor. In particular:
- https://github.com/Originate/exo-add/blob/master/appveyor.yml: the config file
- https://github.com/Originate/exo-add/blob/master/bin/exo-add.cmd: the CLI command for Windows
- https://github.com/Originate/exo-add/blob/master/bin/spec.cmd: the test command. Note how you have to call other `cmd` or `bat` file using `call`, otherwise you delegate execution to them, and the whole script stops when your called file ends.

**Pull request merged**